src/main/java/com/intel/gkl/pdhmm/IntelPDHMM.java

```diff
@@ -52,37 +54,52 @@ public boolean load(File tempDir) {
         return true;
     }
 
+    private static void checkArraySize(Object array, int expectedSize, String arrayName)
+            throws NullPointerException, IllegalArgumentException {
```

Comment:
This "throws" clause is unnecessary. NullPointerException and IllegalArgumentException are RuntimeExceptions which means that they do not have to be declared in "throws" section.